Merge alarm, repeat and details tabs

st3iny opened the pull request:
Fix #3291 

Move the content of the alarm and repeat tab to the details tab. The alarms integrate very well visually. However, the repeat tab stand out because it is not divided into rows like the rest of the details tab.

@nimisha-vijay @jancborchardt Your feedback is appreciated (especially about the repeat section).

![Screenshot 2021-10-01 at 16-41-03 Woche 39 aus 2021 - Kalender - Nextcloud](https://user-images.githubusercontent.com/1479486/135639052-39879944-24d4-4e8d-b793-9f2a856fd89d.png)

nimisha-vijay commented:
Looks super nice actually! I think merging these into one section is a step in the right direction :)
The reminders part looks quite nice, my only feedback is to have the "Add new reminder" at the bottom instead of the top, and remove the left icons in everything except the first entry.

As for the repeat, it looks like a completely separate section, so we can try making it look more similar to the other fields. How about as a starting point, we can show the repeat icon on the left and only the summary, with an edit button on the right, and when you click on the edit button you see all these extra details. If the event is not repeating, then instead of the summary, it's a button that says "Create repeating event" or something similar.

Something like this is what I mean:
![image](https://user-images.githubusercontent.com/52440189/135826253-37595554-2135-4435-8731-4ca7f73a789f.png)
![image](https://user-images.githubusercontent.com/52440189/135826290-26d2d4c9-3bb4-4369-9e44-1a006789724d.png)


What do you think? cc @jancborchardt

jancborchardt commented:
Nice work @st3iny and great feedback @nimisha-vijay! I was thinking about adding a heading for "Reminders", but your proposal is much better. (And we can simplify other parts in the future too :)

st3iny commented:
Thanks for the feedback :) I redesigned both sections and updated the screenshot above. I also added a little screen recording of editing the recurrence.

ChristophWurst commented:
![Bildschirmfoto von 2021-10-12 17-45-38](https://user-images.githubusercontent.com/1374172/136988004-5aa518ef-915b-458b-a66e-c2354c95037c.png)

in an earlier iteration there was breathing space and a little header for the repeating section (and the reminders?). I miss this in the latest version. It's not possibly to visually see what belongs together. Some spacing breaks up the controls into semantic groups. 

I think the text of reminders and recurrence is too dark. Should it be a bit brighter like the controls above?

ChristophWurst commented:
Got it. If this is the intended design then ignore my input :)

ChristophWurst commented:
@GretaD please review and test. If you're okay with it rebase, squash and merge :)
